graspy/cluster/gclust.py

```diff
@@ -1,29 +1,32 @@
 import numpy as np
+import pandas as pd
 from sklearn.metrics import adjusted_rand_score
 from sklearn.mixture import GaussianMixture
 from sklearn.utils.validation import check_is_fitted
+from sklearn.model_selection import ParameterGrid
 
 from .base import BaseCluster
 
 
 class GaussianCluster(BaseCluster):
     r"""
     Gaussian Mixture Model (GMM)
-
     Representation of a Gaussian mixture model probability distribution. 
     This class allows to estimate the parameters of a Gaussian mixture 
     distribution. It computes all possible models from one component to 
     max_components. The best model is given by the lowest BIC score.
-
     Parameters
     ----------
+    min_components : int, defaults to 1. 
+        The minimum number of mixture components to consider (unless
+        max_components=None, in which case the parameter is one the highest
+        number of mixture components to consider and must be less than
+        max_components).
     max_components : int, defaults to 1.
         The maximum number of mixture components to consider.
-
     covariance_type : {'full' (default), 'tied', 'diag', 'spherical'}, optional
-        String describing the type of covariance parameters to use.
-        Must be one of:
-
```

Comment:
Add a newline here.

Reply:
is this to separate required parameters from optional ones?

Reply:
just a bit confused as to where to add it.

Reply:
After `If a string, it must be one of:` and before `- 'full'`